libs/@local/graph/store/src/entity/store.rs

```diff
@@ -433,7 +434,7 @@ pub trait EntityStore {
         actor_id: ActorEntityUuid,
         consistency: Consistency<'_>,
         params: Vec<ValidateEntityParams<'_>>,
-    ) -> impl Future<Output = HashMap<usize, EntityValidationReport>> + Send;
+    ) -> impl Future<Output = Result<HashMap<usize, EntityValidationReport>, Report<QueryError>>> + Send;
```

Comment:
`validate_entity` is allowed to fail if the actor is invalid. This was not covered previously. The validation itself will still always return a proper structure.